generated/ruby/grpc-google-monitoring-v3/Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
```

Comment:
In Ruby, gRPC wouldn't be an independent package but be a part of Gapic client package. Please remove them.